android/source/VideoLocker/src/org/edx/mobile/view/dialog/EnrollmentFailureDialogFragment.java

```diff
@@ -50,7 +50,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         dialog_tv_1.setText(getArguments().getString("dialog_msg_1"));
         // Watch for button clicks.
         Button positiveBtn = (Button) v.findViewById(R.id.positiveButton);
-        positiveBtn.setText(getArguments().getString("dialog_yes_btn"));
+        positiveBtn.setText(getArguments().getString("dialog_yes_btn").toUpperCase());
```

Comment:
Instead, use <code>android:textAllCaps</code> property for the <code>TextView</code> in XML.  This is good, if this dialog class is supposed to show some other contents where buttons do not have text in ALL Caps.